### PR TITLE
Utility DB date range filtering

### PR DESCRIPTION
**Pull request by Mykhailo-Pavliuk**

### CODE REVIEW

File: mainwindow.cpp [hunk 1]
```diff
@@ -31,6 +31,7 @@ MainWindow::MainWindow(QWidget *parent)
     schedule.groupSchedule = groupSchedule;
     utilityDb -> insertScheduleToTable("KN_31", schedule);
     Schedule schedule1 = utilityDb ->getScheduleByTableName("KN_31");
```
Comment (scrooge1987) [inline mainwindow.cpp:33]: Why do use this style "KN_31" but not "KN-31" directly?

File: utilitydb.cpp [hunk 2]
```diff
@@ -5,78 +5,105 @@ UtilityDB::UtilityDB() { establishConnection(); }
 bool UtilityDB::isConnected() { return db.isOpen(); }
```
Comment (scrooge1987) [inline utilitydb.cpp:5]: Do these methods make sense? There is only one return statement and that's all.

**Review by scrooge1987 — CHANGES_REQUESTED** [review]
Please pay attention to my comments, if everything is OK, then I approve.

File: utilitydb.cpp (hunk 2 above)
Comment (Mykhailo-Pavliuk) [reply utilitydb.cpp:5]: And how will you check on the start, whether connection is established?

File: mainwindow.cpp (hunk 1 above)
Comment (Mykhailo-Pavliuk) [reply mainwindow.cpp:33]: Seriously?) Name of table can’t consist of “-“ in SQLite

Comment (scrooge1987) [reply mainwindow.cpp:33]: Ohh, it's a new discovery for me, really..

File: utilitydb.cpp (hunk 2 above)
Comment (scrooge1987) [reply utilitydb.cpp:5]: I must learn how to communicate in English. Really, sorry… Here I thought that this method we use only in if statements, so can we just write if(db.isOpen())?

Comment (scrooge1987) [reply utilitydb.cpp:5]: I am  ~~a piece of dog shit~~, approve.

**Review by scrooge1987 — APPROVED** [review]

**Pull request merged**